### DISCUSSION TITLE
Progress on Trip Calculator

### DISCUSSION THREAD

**Discussion by gh44922023** [gh44922023/Tafe230221#8] [General]
@jeffrey-tafe 
Just checking on progress with "Trip Calculator" Development.
Any issues to report?
Any screenshots to share?

**Comment by jeffrey-tafe** [comment]
@gh44922023 
Trip calculator is complete and pushed to the development branch. Screenshots:

Graph after merge
![image](https://github.com/gh44922023/Tafe230221/assets/141377378/facf6e34-ffaf-4024-8939-56821c6d8a14)

Log after merge
![log-after-merge](https://github.com/gh44922023/Tafe230221/assets/141377378/a25d5eb6-64a4-435c-b566-c79c463f1016)

Main Menu after merge
![main-menu-after-merge](https://github.com/gh44922023/Tafe230221/assets/141377378/6c86965f-33c2-4163-8b49-318a7a27ad6d)

Trip Calculator after merge
![trip-calculator-after-merge](https://github.com/gh44922023/Tafe230221/assets/141377378/ad7b0b77-03b6-40f2-90d6-f6eba9b13cd6)

**Reply by gh44922023** [reply]
Good work Jeff!!!  Job well done.  Were there any conflicts to report?

**Reply by jeffrey-tafe** [reply]
No conflicts :)
